README.md

```diff
@@ -25,6 +25,17 @@ File compressedFile = await FlutterNativeImage.compressImage(file.path,
 You have to give it a file from the file system and optionally provide a quality (1-100) and a resizing percentage (1-100).
 Each platform will use it's proper tools to handle the resizing.
 
+To resize the image to the certain size, use following code:
+```dart
+ImageProperties properties = await FlutterNativeImage.getImageProperties(file.path);
+File compressedFile = await FlutterNativeImage.compressImage(file.path, quality: 80, width: 600, height: 300);
+```
+Keep aspect ratio of the file:
+```dart
+ImageProperties properties = await FlutterNativeImage.getImageProperties(file.path);
```

Comment:
Nitpick: can you make a line break on this line, so the reader does not have to scroll sideways?